BiaDd/hoohacks2022
# Login register

## Status

- Done

## GitHub Issues addressed

- This PR closes #10 

## What I did

- Implemented basic Login functionality
- Implemented basic Signup functionality

## Screenshots

- Before
- After

## Tests

- Go to localhost:3000/signup
- Create a new user, but first try to submit with passwords that don't match
- Now, enter everything properly and submit, you should be taken to localhost:3000/login
- Go back to the signup page, try signup up with the same username or email, you should be prevented from doing so (and an error should be output to the console)
- Now, go to the login page and enter the same credentials as before. You should be logged in and redirected to the home page.

## Questions/Discussions/Notes

-

## Merging the PR

- Who should merge this PR?
  - [] The last reviewer to approve can merge it
